### DISCUSSION TITLE
Tidak bisa mengirimkan gambar QR Code

### DISCUSSION THREAD

**Discussion by rickyok** [kirimwa-id/forum#173] [Tanya Jawab]
Saya menggunakan QR code generator dari google charts yang ada docs nya disini
https://developers.google.com/chart/infographics/docs/qr_codes

Saya testing menggunakan gambar ini : 
https://chart.googleapis.com/chart?cht=qr&chs=400x400&chld=H&chl=https%3A//airtable.rav.ninja/pay2/a1e10a8e-a29f-4f13-b91c-d1f727882f22

Tipe nya adalah PNG. Kalau di check mime nya juga image/png. Tetapi tidak bisa terdetect sebagai gambar.
Error message yang dikembalikan dari API adalah
{'message': 'Failed to send message. Error: Invalid image format, allowed format are JPEG, PNG and GIF.'}

Ini sebetulnya mungkin duplikat dari pertanyaan https://github.com/kirimwa-id/forum/discussions/53 tetapi di sana di mark sebagai wontfix.

Kalau diperlukan saya bisa membuat proxy, tetapi sepertinya proxy saya juga tidak dikenali sebagai png, padahal MIME nya sudah image/png. Kalau baca di diskusi forum sebelum nya, dibaca dari HEAD. Ada usul gimana di flask bisa mengembalikan HEAD yang benar?

**Comment by rickyok** [comment]
Barusan saya juga coba buat sendiri untuk QR code generator

untuk test 

https://airtable.rav.ninja/qr?url=https%3A//airtable.rav.ninja/pay2/a1e10a8e-a29f-4f13-b91c-d1f727882f22

ini juga sama , tidak bisa di gunakan untuk send image

**Reply by akbarism** [reply]
ini sama kaya kasus saya mas, ketika nyimpen gambar di firebase,
solusinya harus disimpen di storage yang ketika di get urlnya, diblakang urlnya harus ada mime imagenya (.jpg/.png/dll)

jadi kayaknya mas ketika generate qr nya harus disimpen dulu ke storage tersebut lalu diambil urlnya, baru dikirim lewat wa
